benchmarks/bench_auto_batching.py

```diff
@@ -0,0 +1,112 @@
+"""Benchmark batching="auto" on high number of fast tasks
+
+The goal of this script is to study the behavior of the batch_size='auto'
+and in particular the impact of the default value of the
+joblib.parallel.MIN_IDEAL_BATCH_DURATION constant.
+
+"""
+# Author: Olivier Grisel
+# License: BSD 3 clause
+
+import numpy as np
+import time
+import os
+from pprint import pprint
+from joblib import parallel, Parallel, delayed
+
+here = os.path.dirname(__file__)
+MEMMAP_FILE = os.path.join(here, '_benchmark_data_file.bin')
```

Comment:
Is there a reason we can not create the .bin in a temporary folder rather than in joblib/benchmarks ? This would allow to remove the *.bin to .gitignore as well.